test/test-labels.js

```diff
@@ -168,31 +184,31 @@ describe('Labels', function () {
     });
   });
 
-  describe('localizeImage', function () {
-    this.timeout(100000);
-
-    it('should take an imageId and a labelId', async function () {
-      const res = await this.api.localizeImage(EVERYDAY_OBJECT_ID, 'cat', {
-        update: true,
-        imageId,
-        labelId,
-      });
-
-      expect(res.results).to.be.an('Array', JSON.stringify(res));
-      expect(res.results).to.have.lengthOf(1, JSON.stringify(res));
-    });
-
-    it('should take an array of imageIds and a labelId', async function () {
-      const res = await this.api.localizeImage(EVERYDAY_OBJECT_ID, 'cat', {
-        update: true,
-        imageIds: [imageId, 'invalid-id'],
-        labelId,
-      });
-
-      expect(res.results).to.be.an('Array', JSON.stringify(res));
-      expect(res.results).to.have.lengthOf(1, JSON.stringify(res));
-    });
-  });
+  // describe('localizeImage', function () {
```

Comment:
Will this be removed or just commented out for now?

Reply:
I was running into an issue where, after creating a detector through the API, localizing an image wouldn't work because `outputToHumanLabels` wasn't populated on the detector and the workflow in `routes/public_api/localize.js` relies on it to determine which label to localize. Not sure if that's the intended behavior or not. @john-matroid do you have any thoughts?